package.json

```diff
@@ -31,47 +31,46 @@
   "homepage": "https://github.com/esbb48/Kizuna#readme",
   "dependencies": {
     "babel-polyfill": "^6.9.1",
-    "grommet": "^0.6.10",
+    "extract-text-webpack-plugin": "^v2.0.0-beta.4",
+    "grommet": "^1.0.1",
     "react": "^15.3.0",
     "react-dom": "^15.3.0",
-    "react-hot-loader": "^3.0.0-beta.1",
+    "react-hot-loader": "^3.0.0-beta.6",
```

Comment:
The reason I think production code definitely doesn't need react-hot-loader. But I saw that `` react-hot-loader `` is used in `` src/index.js ``, looks like it'd be a better idea to have different index file for dev and production.